app/controllers/projects_controller.rb

```diff
@@ -1,6 +1,8 @@
 class ProjectsController < ApplicationController
   def index
-    @projects = Project.includes(:categories).active
+    @projects = Project.paginate(page: params[:page],
+    							 per_page: 10).
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Tab detected.